singer_sdk/helpers/capabilities.py

```diff
@@ -47,6 +48,13 @@
         description="The max depth to flatten schemas.",
     ),
 ).to_dict()
+TARGET_SCHEMA_CONFIG = PropertiesList(
+    Property(
+        "default_target_schema",
+        StringType(),
+        description="The Default schema to place all streams",
```

Comment:
Small wording tweak suggested here, mostly because the word 'schema' is overloaded and can sometimes mean 'JSON schema'.

```suggestion
        description="The default schema name to use for all streams",
```

Reply:
What do you think about  `description="The default target database schema name to use for all streams",`

Reply:
Yeah, sounds great!